lms/djangoapps/labster_course_license/utils.py

```diff
@@ -89,69 +103,84 @@ def get_parent_unit(xblock):
             return xblock
 
 
-class XBlockInfo(object):
-    def __init__(self, info=None):
-        self.is_hidden = info.is_hidden if info else True
-        self.children = list(info.children) if info else []
-
-    def set_visibility(self, value):
-        if self.is_hidden is not False:
-            self.is_hidden = value
-
-
-def get_xblock_info(xblock, course_info, is_hidden=True, child=None):
+def get_block_course_key(block):
+    """
+    Returns course key depending on block id type.
+    """
+    # The incoming block might be a CourseKey instance of some type, a
+    # UsageKey instance of some type, or it might be something that has a
+    # location attribute.  That location attribute will be a UsageKey
+    identifier = getattr(block, 'id', None)
+    if isinstance(identifier, CourseKey):
+        course_key = block.id
```

Comment:
Why don't just return `course_key` in each if? Then we don't need `else: return None`